### PR TITLE
[OSGi] Add Karaf Archive (KAR)

### PR DESCRIPTION
**Pull request by m4rkmckenna**
* Moved unmanaged jars from `vcloud-director/lib` to a maven repo @ `unmanaged-third-party`

### CODE REVIEW

**Comment by neykov** [comment]
LGTM. I don't have the permissions to merge this.

**Comment by m4rkmckenna** [comment]
Thanks @neykov ... @aledsage can you merge if your ok with this?

File: brooklyn-networking-features/pom.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.brooklyn.networking</groupId>
+    <artifactId>brooklyn-networking-features</artifactId>
+    <version>0.10.0-SNAPSHOT</version>  <!-- BROOKLYN_VERSION -->
+    <packaging>kar</packaging>
+    <name>Brooklyn Advanced Networking :: Features</name>
+
+    <properties>
+        <brooklyn.version>0.10.0-SNAPSHOT</brooklyn.version>
```
Comment (aledsage) [inline brooklyn-networking-features/pom.xml]: Add the marker `<!-- BROOKLYN_VERSION -->` at the end of this line.

File: brooklyn-networking-features/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.brooklyn.networking</groupId>
+    <artifactId>brooklyn-networking-features</artifactId>
+    <version>0.10.0-SNAPSHOT</version>  <!-- BROOKLYN_VERSION -->
```
Comment (aledsage) [inline brooklyn-networking-features/pom.xml]: Instead use the marker `<!-- ADVANCED_NETWORKING_VERSION -->`

**Comment by aledsage** [comment]
Thanks @m4rkmckenna - LGTM; merging.

**Pull request merged**